corehq/apps/custom_data_fields/static/custom_data_fields/js/custom_data_fields.js

```diff
@@ -47,7 +47,7 @@ hqDefine('custom_data_fields/js/custom_data_fields', [
             return self.label() || self.slug();
         });
 
-        if (!toggles.toggleEnabled('REGEX_FIELD_VALIDATION')) {
+        if (!initialPageData.get('can_view_regex_field_validation')) {
```

Comment:
It looks like this is the only usage of `toggles` in this file, so you can also remove the dependency from the top of the file.

Have y'all thought about making privileges available to javascript similarly to feature flags, so you wouldn't need to add initial page data? I don't feel strongly about this, but it might be worthwhile if we end up using this feature flag removal pattern a lot.

Reply:
cool, thank you!

Reply:
Indeed there is one, you can use `request_has_privilege` tag, see for e.g. [here](https://github.com/dimagi/commcare-hq/blob/b974e57d0dd6783d2464587ec0078814d7bc2ac9/corehq/apps/users/templates/users/edit_commcare_user.html#L142)

Reply:
I think the `request_has_privilege` is different from what Jenny is referring to. Specifially, it's for the ability to access privileges within JavaScript files similar to how toggles are done with the `toggles.toggleEnabled`. This work would be outside the scope of this PR, so I have created a separate ticket for it [here](https://dimagi-dev.atlassian.net/browse/SC-2610) so that we can address it at some point.